### PR TITLE
[IMP] account: add performance test on account invoice send wizard

### PR DESCRIPTION
**Pull request by tde-banana-odoo**

### CODE REVIEW

**Comment by tde-banana-odoo** [comment]
@qdp-odoo @smetl FYI: adding some performance tests for account invoice send wizard, as we plan to improve the mail composer. I bet there is room to improve the accounting one when working on mail :) .

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Review by qdp-odoo — COMMENTED** [review]
Dude, I don't have any problem reviewing your branches but self r+ instantaneously after the ping is kinda gross. :point_up: 

About the content: I've no real technical remarks, I just wonder if the invoice's mail composer is so customized in account that it requires its own perf test.

**Comment by tde-banana-odoo** [comment]
:cowboy_hat_face:

**Comment by tde-banana-odoo** [comment]
Hello @smetl ,

Thanks for your feedback. Indeed this was a bit hastily r+'ed. Purpose was more to have a basis for some refactoring currently done in mail composer, allowing to have both something already present in master to see its evolution, and have only query count updates in the refactoring branch. However this could have waited for some review obviously.

Account has probably the most custom usage of mail and having more tests is necessary. Composer itself still has no unit tests, hence adding some of them.

If you have some comments I will apply them in master like every review. Purpose of those tests is to asses current behavior of account specific composer based on mail composer (who is under cleaning) as well as some batch-related features of mail. This is not related to accounting specific flows but to mail usage in accounting, hence specific tests setup and naming.

In short: sorry for quick r+, and if you have comments be sure I will apply them :) .

For the peace and love of our two teams !
